engine/src/test/scala/org/scalarules/dsl/core/DatumTestsBerekening.scala

```diff
@@ -0,0 +1,23 @@
+package org.scalarules.dsl.core
+
+import org.scalarules.dsl.core.DatumTestGlossary._
+import org.scalarules.dsl.nl.grammar._
+
+class DatumTestsBerekening extends Berekening (
+
+  Gegeven (DatumA < "01-01-2015".datum)
+    Bereken KleinerDan is "success"
```

Comment:
I understand the use of KleinerDan and co as the equivalent of the mathematical comparator, but it just looks weird in Dutch. How about using "EerderDan" etc? We are trying to write a DSL afterall. :)